src/assets/wise5/components/graph/graphService.ts

```diff
@@ -492,4 +496,27 @@ export class GraphService extends ComponentService {
     }
     return series;
   }
+
+  getClassmateStudentWork(
+    nodeId: string,
+    componentId: string,
+    periodId: number,
+    showWorkNodeId: string,
+    showWorkComponentId: string,
+    showClassmateWorkSource: string
+  ) {
+    const runId = this.configService.getRunId();
+    switch (showClassmateWorkSource) {
+      case 'period':
+        return this.http.get(
+          `/api/classmate/graph/student-work/${runId}/${nodeId}/${componentId}/${showWorkNodeId}/${showWorkComponentId}/period/${periodId}`
+        );
+      case 'class':
+        return this.http.get(
+          `/api/classmate/graph/student-work/${runId}/${nodeId}/${componentId}/${showWorkNodeId}/${showWorkComponentId}/class`
+        );
+      default:
+        throw $localize`Invalid Show Classmate Work Source`;
```

Comment:
Should this block instead be an if/else since we're only allowing 'period' or 'class'?